src/app/ApplicationTemplate.UWP/Views/Styles/Controls/TextBlock.xaml

```diff
@@ -230,20 +165,10 @@
 
 	<Style x:Key="ButtonText"
```

Comment:
please just make sure that the changes you bring in this style are actually applied to the button styles